src/pkg.jl

```diff
@@ -94,18 +94,27 @@ end
 
 get_snoopfile(pkg::Types.PackageSpec) = get_snoopfile(root_path(pkg))
 
+const relative_snoop_locations = (
+    "snoopfile.jl",
+    joinpath("snoop", "snoopfile.jl"),
+    joinpath("test", "snoopfile.jl"),
+    joinpath("test", "runtests.jl"),
+)
```

Comment:
This list seems a little inflexible. I wanted to collect my snoopfile and a script to call PackageCompiler in a directory of my choice, say `compile`. If `relative_snoop_locations` had been a vector instead of a tuple I could have modified it as needed, but maybe there should be a more direct API to specify such things.

Reply:
Creating your own non package specific snoop files is a pretty different task so will need a different api (or at least a different function to call).
E.g. resolving packages etc will need to work much differently

Reply:
Really? I made a shortened version of my `test/runtests.jl` and it seemed to work fine as a snoop file. It was very much package specific, just that I wanted the naming differently.